.github/workflows/e2e_miscellaneous_tests.yml

```diff
@@ -63,6 +63,11 @@ jobs:
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite="contributorDashboard" --prod_env
         env:
           VIDEO_RECORDING_IS_ENABLED: 0
+      - name: Run Mobile Contributor Dashboard Tests
+        if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
+        run: xvfb-run -a --server-args="-screen 0, 600x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite="contributorDashboard" --mobile --prod_env
```

Comment:
Any specific reason for why you have chosen 600x1000x24?

Reply:
I have chosen 600px as the width to ensure all contributor pages switch to the mobile view.

Reply:
@SAEb-ai Is it important to have the screen size mentioned in `server-args` as well (along with config file)?